05-onchain.md

```diff
@@ -523,67 +523,70 @@ irrevocably resolved, should still protect against this happening. [ FIXME: May
 ## Penalty Transactions Weight Calculation
 
 There are three different scripts for penalty transactions, with the following
-witnesses weight (details of the computation are in
+witnesses weights (details of weight computation are in
 [Appendix A](#appendix-a-expected-weights)):
 
     to_local_penalty_witness: 160 bytes
     offered_htlc_penalty_witness: 243 bytes
     accepted_htlc_penalty_witness: 249 bytes
 
-The penalty txinput itself takes 41 bytes, which has a weight of 164, making the
-weight of each input:
+The penalty *txinput* itself takes up 41 bytes and has a weight of 164 bytes,
+which results in the following weights for each input:
 
     to_local_penalty_input_weight: 324 bytes
     offered_htlc_penalty_input_weight: 407 bytes
     accepted_htlc_penalty_input_weight: 413 bytes
 
-The rest of the penalty transaction takes 4+1+1+8+1+34+4=53 bytes of non-witness
-data, assuming that it has a pay-to-witness-script-hash (the largest standard output
-script), in addition to a 2-byte witness header.
+The rest of the penalty transaction takes up 4+1+1+8+1+34+4=53 bytes of
+non-witness data: assuming it has a pay-to-witness-script-hash (the largest
+standard output script), in addition to a 2-byte witness header.
 
-In addition to outputs being swept under as penalty, the node MAY also sweep the
+In addition to the outputs being swept under [ FIXME: what does 'swept under' mean? ] as penalty, the node MAY also sweep the
```

Comment:
Yeah, "under" here is wrong.  Swept is a term for spending an entire collection of outputs (usually the context is moving from one wallet to another, by the same user).  Rework this?

Here's my stab at it:

    In addition to spending these outputs, a penalty transaction may optionally spent the `to_remote` output (e.g. to reduce the total amount paid in fees). Doing so...

Reply:
changed